wordpress-mobile/WordPress-iOS
# Posts & Pages: Move post actions to a context menu

Closes #21717 

## Description
- Converted the Posts action sheet to a context menu
- Post actions are divided into sections

Before | After
-- | --
<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/2c043430-f6af-4ebe-b6e6-11201f7c39f7" width=200> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/8f031d4e-868f-4bc5-91c7-0d2f0247cfd5" width=200>

## Notes
- TODO
  - Add comments action
  - Adapt for Pages context menu
- Using system images for the icons, for consistency - I can change this if needed

## How to test

- On the Posts list page, tap on the ellipsis button
- Verify post actions are presented in a context menu
- Verify post actions are in the correct grouping (ref: eYeHXEMDbnFptE40xUqZ2T-fi-836_5909)
- Verify each post action has the expected result (no regressions)

## Regression Notes
1. Potential unintended areas of impact
Post actions

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
